plugin.json

```
@@ -1,8 +1,11 @@
 {
     "id": "autotranslate",
     "name": "Autotranslate",
-    "description": "Autotranslate plugin for Mattermost 5.4 and later.",
-    "version": "0.2.0",
+    "description": "Autotranslate plugin for Mattermost 5.22 and later.",
+    "homepage_url": "https://github.com/mattermost/mattermost-plugin-autotranslate",
```

https://github.com/mattermost/mattermost-plugin-autotranslate/pull/27#discussion_r429870396 was a bad suggestion, sorry.
```suggestion
    "homepage_url": "https://github.com/mattermost/mattermost-plugin-autotranslate",
    "support_url": "https://github.com/mattermost/mattermost-plugin-autotranslate/issues",
```